lib/AppInfo/Application.php

```diff
@@ -64,6 +67,19 @@ public function register(IRegistrationContext $context): void {
 	public function boot(IBootContext $context): void {
 		Util::connectHook('OC_Filesystem', 'preSetup', $this, 'addStorageWrapper');
 
+		$eventDispatcher = $context->getServerContainer()->get(IEventDispatcher::class);
+		$eventDispatcher->addListener('OCA\DAV\Connector\Sabre::addPlugin', function (SabrePluginEvent $event) use ($context) {
```

Comment:
this seems to only work on `remote.php/dav/admin/files/` it does not block `remote.php/webdav/`

Right?

Reply:
Ignore that,  had some temporary code in place

Reply:
I just tried and it actually blocks `remote.php/webdav/` requests as well.